index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css"
+        integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <title>Full stack software Engineer</title>
+</head>
+
+<body>
+    <section class="background-image">
+        <header>
+            <nav class="nav">
+                <div class="container d-flex justify-content-between">
+                    <div class="my-logo text--primary weight--700">My Logo</div>
```

Comment:
- [ ] Kindly make sure your `Logo` is grabbed inside an anchor tag and not a generic `div` that will be redirecting the user to the home page for a better UX as it is the standard.
`Hint: <a href="#">Logo</a>`